### PR TITLE
symfonise notifications' save

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

**Review by alexarn — CHANGES_REQUESTED** [review]
You add changes from an other pull request (298) and that have been merged on master already. Why ?
Also, provide a config/permissions.yaml with only the newly added routes

2nd option:
rebase your branch on master

File: config/permissions.yaml [hunk 1]
```diff
@@ -13,3 +13,7 @@
     - 100
 "ajax.holidaycredit":
     - 100
+"notifications.index":
```
Comment (alexarn) [inline config/permissions.yaml:16]: Good thing to provide notifications.index. So you can remove the entry in access with page = notification

File: src/Controller/NotificationController.php [hunk 2]
```diff
@@ -9,6 +9,7 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
+require_once(__DIR__ . '/../../public/include/config.php');
```
Comment (alexarn) [inline src/Controller/NotificationController.php:12]: Why do you need config.php here ?

**Review by alexarn — CHANGES_REQUESTED** [review]

File: src/Controller/NotificationController.php (hunk 2 above)
Comment (sillydebs) [reply src/Controller/NotificationController.php]: I cannot remember, sorry.

File: config/permissions.yaml (hunk 1 above)
Comment (sillydebs) [reply config/permissions.yaml:16]: Thank you. db_data.php and atomicupdate are now updated.

File: config/permissions.yaml [hunk 3]
```diff
@@ -13,3 +13,7 @@
...
+    - 21
+"notifications.save":
+    - 21
```
Comment (alexarn) [inline config/permissions.yaml:19]: Routes names are wrong in the permissions.yaml.
In the controller, the are called notification.xxxx
So access denied

**Review by alexarn — CHANGES_REQUESTED** [review]
Error when saving notifications => Access denied

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**